full-service/src/json_rpc/wallet_api.rs

```diff
@@ -0,0 +1,880 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+//! Wallet API.
+//!
+//! A [JSON RPC 2.0](https://www.jsonrpc.org/specification) API inspired by [Stratum](https://github.com/aeternity/protocol/blob/master/STRATUM.md) .
```

Comment:
I don't agree that this was inspired by stratum - it was inspired by the [BEAM Wallet API](https://github.com/BeamMW/beam/wiki/Beam-wallet-protocol-API), which references stratum for structure, but BEAM's wallet API was more directly the influence for this API.